examples/spanner-interleave-subquery/README.md

```diff
@@ -0,0 +1,23 @@
+# spanner-interleave-subquery
+
+This example contains the benchmark code to examine how efficient we can query for Cloud Spanner interleaved tables with subqueries.
+
+## Prerequisite
+
+Create a Cloud Spanner database with [schema.sql](schema.sql).
```

Comment:
Make this easy for the user.
Please add a gcloud command or better yet a small terraform module to create this spanner instance.

Reply:
Added gcloud command. I think adding terraform module would be a little bit heavy for this kind of one-off benchmark code.